doc/lua_api.txt

```diff
@@ -1310,7 +1310,24 @@ factor!
 The `z_index` field specifies the order of HUD elements from back to front.
 Lower z-index elements are displayed behind higher z-index elements. Elements
 with same z-index are displayed in an arbitrary order. Default 0.
-Supports negative values.
+Supports negative values. By convention, the following values are recommended:
+
+*  -400: Graphical effects, such as vignette
+*  -300: Name tags, waypoints
+*  -200: Wieldhand
+*  -100: Things you wear on your face or stuff that is blocking your view,
+         such as a mask
+*     0: Default. For builtin HUD elements and anything that is
+         considered to be the “true” HUD:
+         Minimap, hotbar, builtin statbars, crosshair
```

Comment:
How about:
`Standard in-game HUD elements that are usually visible: Minimap, hotbar...`